<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Neringa-PC" name="tests.Tmp" tests="5" failures="0" timestamp="9 May 2020 11:04:24 GMT" time="0.035" errors="1">
  <testcase name="assertTrue" time="0.002" classname="tests.Tmp"/>
  <testcase name="assertFalse" time="0.003" classname="tests.Tmp"/>
  <testcase name="assertNull" time="0.010" classname="tests.Tmp"/>
  <testcase name="assertEquals" time="0.017" classname="tests.Tmp">
    <error type="org.junit.ComparisonFailure" message="a expected:&lt;[b]&gt; but was:&lt;[tikejosi a]&gt;">
      <![CDATA[org.junit.ComparisonFailure: a expected:<[b]> but was:<[tikejosi a]>
	at org.junit.Assert.assertEquals(Assert.java:125)
	at tests.Tmp.assertEquals(Tmp.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- assertEquals -->
  <testcase name="assertNotNull" time="0.003" classname="tests.Tmp"/>
</testsuite> <!-- tests.Tmp -->
